packages/editor-ui/src/stores/assistant.store.ts

```diff
@@ -438,6 +439,30 @@ export const useAssistantStore = defineStore(STORES.ASSISTANT, () => {
 			context: visualContext,
 			question: userMessage,
 		};
+		const payloadSize = getObjectSizeInKB(payload);
+		if (payloadSize > AI_ASSISTANT_MAX_CONTENT_LENGTH) {
```

Comment:
after we remove all these params, object parameter might still be too big. Should not we check again or remove more things or fail with a clear error to the user?

Reply:
Definitely, I think the chance is slim but we should account for it to complete the solution. Will add that.